tests/flagd/pkg/integration/integration.go

```diff
@@ -15,11 +15,12 @@ type ctxStorageKey struct{}
 
 // ctxClientKey is the key used to pass the openfeature client across context.Context
 type ctxClientKey struct{}
+var domain = "flagd-e2e-tests"
```

Comment:
This will still use the same domain/provider between runs, but it's an improvement.

You could attempt to inject a different domain each run, either as a param somehow or through the Go context, maybe?

Reply:
unfortunately, this is not straightforward. We register a provider supplier, which is a one-time operation (supplier yields a new provider ). And there's no way to customize this unless we do this per test invoke, which means a change in gherkin test defintion 🤔

Reply:
I will resolve this conversation with above jutification :)